.wiki/using-wp-ultimo-affiliatewp-integration-to-create-your-own-referral-program-for-wp-ultimo.md

```
@@ -1,20 +1,20 @@
 First of all, you **need to have** [**AffiliateWP**](https://affiliatewp.com/) **active**! You won't be able to use our add-on if you don't have **the plugin that integrates with it**. In case you don’t plan to let your clients use it on their sites as well, AffiliateWP should be active only on your main site.
 
-Also, you’ll need both **WP Multisite WaaS** and **WP Multisite WaaS: AffiliateWP** **activated** **network-wide**.
+Also, you’ll need both **Multisite Ultimate** and **Multisite Ultimate: AffiliateWP** **activated** **network-wide**.
 
-## Adding WP Multisite WaaS as an AffiliateWP Integration
+## Adding Multisite Ultimate as an AffiliateWP Integration
 
-On your main site, head to Affiliates Settings, and on the Integrations tab, check WP Multisite WaaS. This will tell [AffiliateWP](https://affiliatewp.com/) to start tracking visits and conversions from the WP Multisite WaaS sign-up flow.
+On your main site, head to Affiliates Settings, and on the Integrations tab, check Multisite Ultimate. This will tell [AffiliateWP](https://affiliatewp.com/) to start tracking visits and conversions from the Multisite Ultimate sign-up flow.
 
 [![](https://wp-ultimo-space.fra1.cdn.digitaloceanspaces.com/hs-602126536867724dfc6f0a54-zkWh3TFms-1F9087E6-928C-4043-94B9-F85966AF05DA.png)](https://wp-ultimo-space.fra1.cdn.digitaloceanspaces.com/hs-602126536867724dfc6f0a54-zkWh3TFms-1F9087E6-928C-4043-94B9-F85966AF05DA.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)